test/unit/web-server.spec.js

```diff
@@ -180,6 +182,23 @@ describe('web-server', () => {
         })
     })
 
+    it('should load custom handlers', () => {
+      servedFiles(new Set())
+
+      // TODO(vojta): change this, only keeping because karma-dart is relying on it
```

Comment:
Please change or remove this comment as karma-dart is no longer the blocker.

Reply:
Fixed and added a point to https://github.com/karma-runner/karma/issues/3503.